### DISCUSSION TITLE
What observational effects should we include? (Full List)

### DISCUSSION THREAD

**Discussion by bnord** [deepskies/DeepSZSim#53] [Ideas]
which effects should we include?

1. white noise
2. beam (size and shape)

**Comment by bnord** [comment]
issue here: https://github.com/deepskies/SimSZ/issues/42

**Comment by antwinew1401** [comment]
Address not found
Your message wasn't delivered to ***@***.**** because
the address couldn't be found, or is unable to receive mail.
LEARN MORE <https://support.google.com/mail/answer/6596>
The response was:

The email account that you tried to reach does not exist. Please try
double-checking the recipient's email address for typos or unnecessary
spaces. Learn more at https://support.google.com/mail/answer/6596
